docs/home_zn_CH.rst

```diff
@@ -0,0 +1,42 @@
+.. meta::
```

Comment:
The file name should be `home_zh_CN.rst`. 😄 I know what's it like for non-Chinese people.